src/routes/v1/organisations/registrations/summary-logs/upload-completed/post.js

```diff
@@ -143,16 +135,21 @@ export const summaryLogsUploadCompleted = {
     const { summaryLogUpload } = payload.form
 
     try {
-      const status = await upsertSummaryLog(
+      const status = await updateStatusBasedOnUpload(
         summaryLogsRepository,
         summaryLogId,
         summaryLogUpload,
         logger
       )
 
       if (status === SUMMARY_LOG_STATUS.VALIDATING) {
-        const summaryLog = await summaryLogsRepository.findById(summaryLogId)
-        await summaryLogsValidator.validate(summaryLog)
+        const { version, summaryLog } =
+          await summaryLogsRepository.findById(summaryLogId)
```

Comment:
I think this should move into the validator, as it's the async bit. Then we only need to pass the id in (it looks a bit messy right now) rather than the whole summary log